app/scripts/modules/kubernetes/src/v2/manifest/manifest.service.ts

```diff
@@ -12,6 +21,18 @@ export interface IManifestParams {
 
 export type IManifestCallback = (manifest: IManifest) => void;
 
+export function manifestIdentifier(manifest: IStageManifest) {
```

Comment:
v minor nit: It's possibly more Deck-like to include this as a static method on the service class.

Reply:
Updated!

Reply:
Nice. :+1: